cypress/e2e/login/login.spec.ts

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { config } from "dotenv";
+
+import { SynapseInstance } from "../../plugins/synapsedocker";
+import 'dotenv/config';
+
+config({ path: '../../../.env', debug: true });
+describe("Login", () => {
+    let synapse: SynapseInstance;
+
+    beforeEach(() => {
+        cy.visit("/#/login");
+        cy.startSynapse("consent").then(data => {
+            synapse = data;
+        });
+    });
+
+    afterEach(() => {
+        cy.stopSynapse(synapse);
+    });
+
+    describe("m.login.password", () => {
+        // Should be random user on docker
+        const username = Cypress.env('E2E_TEST_USER_EMAIL');
+        const password = Cypress.env('E2E_TEST_USER_PASSWORD');
+        const key = Cypress.env('E2E_TEST_USER_SECURITY_KEY');
+
+        beforeEach(() => {
+            // cy.registerUser(synapse, username, password);
+        });
+
+        it("logs in with an existing account and lands on the home screen", () => {
+            cy.injectAxe();
+
+            cy.get("#mx_LoginForm_username", { timeout: 15000 }).should("be.visible");
+            cy.percySnapshot("Login");
+            // cy.checkA11y();
+
+            cy.get(".mx_ServerPicker_change").click();
+            // cy.get(".mx_ServerPickerDialog_otherHomeserver").type(synapse.baseUrl);
+            cy.get(".mx_ServerPickerDialog_continue").click();
+            // wait for the dialog to go away
+            cy.get('.mx_ServerPickerDialog').should('not.exist');
+
+            cy.get("#mx_LoginForm_username").type(username);
+            cy.get("#mx_LoginForm_password").type(password);
+            cy.startMeasuring("from-submit-to-home");
+            cy.get(".mx_Login_submit").click();
+
+            // TCHAP: Verify device step
+            // this should not be necessary when creating users but for now I get this too often to ignore it
```

Comment:
```suggestion
            // this happens when a user logs in a new device and has other existing devices. (does it happen when there are no other devices ?) 
```

Est-ce que la solution serait de log out de toutes les autres sessions du user ? 

On verra pour une version future, en tout cas cette solution marche.